Servers/Infrastructure/OJS.Servers.Infrastructure/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -159,8 +160,13 @@ public static class ServiceCollectionExtensions
 
             services.AddHangfireServer(options =>
             {
+                var appQueueName = app.ToString().ToLowerInvariant();
+                var queues = handleDefaultQueue
+                    ? new[] { appQueueName, "default" }
```

Comment:
Doesn't Hangfire provide a constant for this?

Reply:
I didn't find one.

Reply:
Now default queue is not added at all